src/Native/src/kernels/stackvm/tensor_ops.cpp

```diff
@@ -385,9 +385,9 @@ result<value_t> nncase::kernels::stackvm::log_softmax(
     try_f32_input(in_mem, input);
     try_f32_output(out_mem, output, input_tensor->shape());
     try_positive_axis(axis_value, axis, input_tensor);
-    try_(reference::softmax(in_mem, out_mem, input_tensor->shape(),
-                            input_tensor->strides(), output_tensor->strides(),
-                            axis_value, 1.f, true));
+    try_(optimized::log_softmax(in_mem, out_mem, input_tensor->shape(),
```

Comment:
Should use contiguous kernel helper as the optimized kernel doesn't handle incontiguous inputs.

Reply:
Done1